charts/internal/e2e-resources/templates/pods-pending.yaml

```diff
@@ -13,10 +13,10 @@ spec:
         limits:
           # Force a 'CreateContainerError':
           # 'Error response from daemon: Minimum memory limit allowed is 6MB'
-          memory: 5M
+          memory: 13M
```

Comment:
Is there a specific reason we are changing this? The reason I ask is that our e2e tests may rely on this number.

Reply:
While testing the e2e-resources, there were instances where the pod kept running out of memory and this was an artifact of that work. i retested it with 5M with no issues so moving it back down. The e2e-tests still passed but no sense making unnecessary changes :)